hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/AcceptanceTestProperties.java

```diff
@@ -90,6 +91,9 @@ public class AcceptanceTestProperties {
     @NotNull
     private Duration startupTimeout = Duration.ofMinutes(60);
 
+    @NotNull
+    private NodeNameEnum nodeType;
```

Comment:
Remove default from `application.yml` and move to here. Also document in config table in `hedera-mirror-test/README.md`.

```suggestion
    private NodeNameEnum nodeType = NodeNameEnum.MIRROR;
```

Reply:
Now it should be okay.

Reply:
Extra semicolon and didn't update docs

Reply:
Removed the extra semicolon and updated the docs.